py/set.go

```diff
@@ -111,6 +111,20 @@ func (s *Set) M__iter__() (Object, error) {
 	return NewIterator(items), nil
 }
 
+func (s *Set) M__and__(other Object) (Object, error) {
+	ret := NewSet()
+	b, ok := other.(*Set)
+	if !ok {
+		return nil, TypeError
```

Comment:
```go
return nil, ExceptionNewf(TypeError, "unsupported operand type(s) for &: '%s' and '%s'", s.Type().Name, other.Type().Name)
```

Reply:
I change the return value of the function